package.json

```diff
@@ -1,16 +1,16 @@
 {
   "name": "specshell",
-  "version": "0.0.2",
+  "version": "0.0.3",
   "description": "Control a program (typically a shell) over standard I/O for testing it",
-  "main": "dist/shell.js",
+  "main": "dist/main/shell.js",
   "scripts": {
-    "build": "tsc",
+    "build": "tsc -b",
     "lint": "tslint -p .",
     "test": "ava -- dist/test/**.js",
     "prepublishOnly": "rm -rf dist/ && npm run build && npm test && npm run lint"
   },
   "files": [
-    "dist/",
+    "dist/main",
```

Comment:
Hmm... more nested folders!

Reply:
I call upon the TypeScript Titans: prove me!
https://github.com/RyanCavanaugh/project-references-demo/blob/9e6bb8eb08e312d11df84ff9d6567af185a2af64/animals/tsconfig.json#L4
```json
{
  "extends": "../tsconfig-base.json",
  "compilerOptions": {
    "outDir": "../lib/animals",
    "rootDir": ".",
  },
  "references": [
    { "path": "../core" }
  ]
}
```

(If you're wondering, that repo just has an incorrect `"main": "index.js"` entry in its `package.json`.  We'd have to go to the Nomic NPM Know-alls to fix _that_...).